src/System.Runtime.CompilerServices.Unsafe/src/Configurations.props

```diff
@@ -4,7 +4,6 @@
     <BuildConfigurations>
       netstandard1.0;
       netstandard;
-      netcoreapp;
```

Comment:
Is the following code now dead with the removal of this line?

https://github.com/dotnet/corefx/blob/master/src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.il#L265-L273

I would have thought we still wanted to build and package for `netcoreapp`, but it would just be OOB.

Reply:
Good point. It will be a minor performance regression otherwise. It should build and packaged for `netcoreapp2.0` that will cover the live netcoreapp as well).

Reply:
Opened https://github.com/dotnet/corefx/issues/26145 so that this is not forgotten.